packages/atomic/.storybook/Introduction.mdx

```diff
@@ -0,0 +1,33 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+
+<Meta title="Introduction" />
+
+# Welcome to the Coveo Atomic Storybook
+This Storybook showcases interactive documentation for the [Coveo Atomic library](https://docs.coveo.com/en/atomic/latest)
+
+Here you can find guides for using Atomic web components for each of the solutions which Coveo offers:
+
+* [Commerce](/docs/commerce-introduction--docs)
+* [Insight](/docs/insight-introduction--docs)
+* [Recommendation](/docs/recommendations-introduction--docs)
+* [Search](/docs/search-introduction--docs)
+
+You can find information about components which are used across all solutions in [Common](/docs/common-introduction--docs).
+
+If you are looking to integrate Coveo with a web framework, Coveo offers wrappers for [React](https://docs.coveo.com/en/atomic/latest/usage/frameworks/atomic-react-wrapper/) and [Angular](https://docs.coveo.com/en/atomic/latest/usage/frameworks/atomic-angular-wrapper/).
+
+## Training
+If you are looking for additional training resources, review the available courses:
+* [Coveo Atomic Tutorial](https://levelup.coveo.com/learn/courses/atomic)
```

Comment:
There are other courses available why just this one?
Also I don't know how often these links change, wouldn't be better to link to a doc page instead where this content can be dynamically updated instead of trying to capture it in markdown here?

For example this one if it's not already used in these pages?
https://docs.coveo.com/en/atomic/